### PR TITLE
feat: updated logs

### PR DESCRIPTION
**Pull request by BenjaminHalko**
closes: #1525

### CODE REVIEW

File: lib/ui/views/installer/installer_viewmodel.dart [hunk 1]
```diff
@@ -212,10 +245,9 @@ class InstallerViewModel extends BaseViewModel {
       'Integration source: ${_managerAPI.getIntegrationsRepo()}',
       
       '\n- Logs',
-      logs,
```
Comment (Ushie) [inline lib/ui/views/installer/installer_viewmodel.dart:215]: Why not put logsTrimmed here instead of appending it in the clipboard copy?

**Review by Ushie — CHANGES_REQUESTED** [review]

File: lib/ui/views/installer/installer_viewmodel.dart (hunk 1 above)
Comment (BenjaminHalko) [reply lib/ui/views/installer/installer_viewmodel.dart:215]: Because the I would have to write .join('\n').

Comment (BenjaminHalko) [reply lib/ui/views/installer/installer_viewmodel.dart:215]: Although it might be cleaner?

Comment (BenjaminHalko) [reply lib/ui/views/installer/installer_viewmodel.dart:215]: Maybe I should do that....

Comment (BenjaminHalko) [reply lib/ui/views/installer/installer_viewmodel.dart:215]: There, changed it.

Comment (Ushie) [reply lib/ui/views/installer/installer_viewmodel.dart:215]: Are you sure you need to join \n here? it's still being joined by the clipboard copy anyways

Comment (Ushie) [reply lib/ui/views/installer/installer_viewmodel.dart:215]: Nvm

**Review by Ushie — APPROVED** [review]

**Pull request merged**